src/Proto.Actor.Extensions/ServiceCollectionExtensions.cs

```diff
@@ -9,7 +8,6 @@ public static class ServiceCollectionExtensions
         public static void AddProtoActor(this IServiceCollection services, Action<ActorPropsRegistry> registerAction = null)
         {
             services.AddSingleton<IActorFactory, ActorFactory>();
-            services.TryAdd(ServiceDescriptor.Singleton(typeof(EventStream<>), typeof(EventStream<>)));
```

Comment:
The DI framework can't resolve EventStream<> as a dependency because there is no public constructor, so removing this line since it has no impact.